mods/sethome/init.lua

```diff
@@ -24,42 +25,58 @@ loadhomes()
 
 minetest.register_privilege("home", "Can use /sethome and /home")
 
-local changed = false
+function sethome.go_home(name)
+	local player = minetest.get_player_by_name(name)
+	if not player then
+		-- just a check to prevent the server crashing
+		return false
+	end
+	if homepos[name] then
+		player:setpos(homepos[name])
+		minetest.chat_send_player(name, "Teleported to home!")
+		return true
```

Comment:
You should return true without sending message, and code the message sending in the chat command callback. Otherwise this is not very useful as a public function.